src/dao.py

```diff
@@ -0,0 +1,82 @@
+import time
+from db import session, ChatStatus, User, Chat, BagelInstance
+
+
+def print_thick_spacer():
+    print('====================')
+
+def print_thin_spacer():
+    print('--------------------')
+
+
+# User
+
+
+def user_print_all(): 
+    print_thick_spacer()
+    print('Users')
+
+    users = User.query().all()
+    for user in users:
+        print(user.name)
+
+    print_thin_spacer()
```

Comment:
not sure how this looks in the console but consider adding new lines in addition to spacers?